### PR TITLE
Update the default value of grpc-raft-conn-num

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Update the default value of `grpc-raft-conn-num` from `10` to `1`.

### Which TiDB version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):https://github.com/pingcap/docs-cn/pull/5669/files#r603843190

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tune-tikv-performance.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ region-split-check-diff = "32MB"
 # RocksDB 能够打开的最大文件句柄数。
 # max-open-files = 40960
 
-# RocksDB MANIFEST 文件的大小限制. # 更详细的信息请参考：https://github.com/facebook/rocksdb/wiki/MANIFEST
+# RocksDB MANIFEST 文件的大小限制 # 更详细的信息请参考：https://github.com/facebook/rocksdb/wiki/MANIFEST
```
Comment (TomShawn) [inline tune-tikv-performance.md:124]: This is to remove a control character.

**Review by sticnarf — APPROVED** [review]

**Review by yikeke — APPROVED** [review]

**Pull request merged**